cmd/lakectl/cmd/annotate.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/treeverse/lakefs/pkg/api"
+)
+
+const annotateTemplate = `{{  $val := .Commit}}
+{{ $.Object|ljust 15}} {{ $val.Committer|ljust 20 }} {{ $val.Id | printf "%.16s"|ljust 20 }} {{ $val.CreationDate|date }}  {{ $.CommitMessage |ljust 30 }}
```

Comment:
how is align works with long object path? how the user knows the output format? consider using a table to render the output.

Reply:
In the case the object name is very long it will at most overlap to the next line. The next line will always start on a new line, so the user can always assume the data is starting there.

Reply:
Got it - regarding the align.
About the second part - how the user knows the output format: I think we need to specify the output format if it is not self explained.